usaon_vta_survey/util/authorization.py

```diff
@@ -0,0 +1,14 @@
+from flask_login import current_user
+
+
+def limit_response_editors() -> None:
+    try:
+        if not (
+            current_user.role_id == 'admin' or current_user.role_id == 'respondent'
+        ):
+            raise RuntimeError(
+                "You must be a respondent or admin to respond to this survey."
+            )
+    except AttributeError as AE:
+        # TODO: Add link to login here?
+        raise RuntimeError("Please login to use this application.") from AE
```

Comment:
```suggestion
    except AttributeError as e:
        # TODO: Add link to login here?
        raise RuntimeError("Please login to use this application.")
```

`e` is a conventional name for a caught error. When raising in an `except` block, we don't need to re-raise `from e` because that behavior is automagical! :) 

That said, do you think it might be more readable to re-arrange the try-except into an `if`?:

```python
if not current_user.is_authenticated():
    raise RuntimeError("Login!")

if current_user.role not in ['admin', 'respondent']:
    raise RuntimeError("Wrong role!")
```

I should have thought of `not in ['X', 'Y']` earlier, I think that part is much more readable than `not (role=='X' or role=='Y')` :)